app/views/shared/_nav.html.haml

```diff
@@ -37,7 +37,7 @@
           - else
             = link_to "Sign in with GitHub", user_github_omniauth_authorize_path
 
-      = form_tag search_path, method: "get", role: "search", class: "navbar-form navbar-right" do
+      = form_tag search_path, method: "get", role: "search", class: "navbar-form navbar-right", id: "searchf" do
```

Comment:
Yes, I meant search form. But I didn't use it because it was used at other places too (probably for some other search bar) and didn't possibly want to mess with other the search bars on the website. 
I am adding another commit with the name nav-search-form to indicate navigation bar search form. But sir if you think that using search-form is better and will not cause any other problems, I will make the change in another commit.